src/csv/compile-csv-feed.ts

```diff
@@ -12,7 +12,7 @@ type Feature = {
 export function compileCsvFeedEntry(
   geojsonFeature: Feature | undefined,
   feedTemplate: CsvDatasetTemplate,
-  feedTemplateTransforms: TransformsList): string {
+  feedTemplateTransforms: TransformsList | undefined): string {
```

Comment:
This works, but you can also add it as an optional since it's the last argument:
`feedTemplateTransforms?: TransformsList`

Reply:
This makes sense and I think it is better since `feedTemplateTransforms` is optional